src/publicapi.js

```diff
@@ -158,6 +158,10 @@ function getInterface(v) {
       if (this.__controller.blurred) this.__controller.cursor.hide().parent.blur();
       return this;
     };
+    _.empty = _.clear = function() {
+      this.select().__controller.backspace();
+      return this;
+    };
```

Comment:
Could be less of a hack than what we did :grinning: 

I think that [accessing the focused field] was one of the things we tried to add in the [StaticMathWithEditables commit](https://github.com/Learnosity/mathquill/commit/5c80199f028f7edd5a1ed5fe389764e0b1601ef0) that @karoltarasiuk  mentioned on our call the other night (that's what `withActiveNode()` is there for).

What we did isn't a great solution - because sometimes you want your method to act on the entire mathquill instance, and other times you want it to affect just the focused field. E.g. I can't just change the `.latex()` method so it only affects the focused bit.